scratch_space/pubspec.yaml

```diff
@@ -1,8 +1,11 @@
 name: scratch_space
-version: 1.0.2-dev
-description: A tool to manage running external executables within package:build
+version: 1.0.2
+description: A tool to manage running external executables within package:build.
 repository: https://github.com/dart-lang/build/tree/master/scratch_space
 
+topics:
+  - build-runner
```

Comment:
I think we have used "code generation" for some other packages, so it might be worth having a bit more general topic here also.

It also feels like maybe this should be `build_runner` since its the name of the package?

Reply:
Topics are supposed to only contain dashes or alphanumeric characters:

https://github.com/dart-lang/site-www/pull/4807/files#diff-96d69c1c40466944f504bd8a24f1f4741020e348d7a360115acaeaa3039708b7R436-R437

Reply:
This package is used by codegen packages, but is it itself codegen related?

Reply:
> Topics are supposed to only contain dashes or alphanumeric characters:

Does that imply we shouldn't be using other packages as topics, given they use underscores?

> This package is used by codegen packages, but is it itself codegen related?

I am not really sure how to draw the line here... yes this code does not itself generate code but it is intended for use in code generators.

Reply:
Hmm, well, this is something we can always change w/ a commit and a publish. I added `'codegen'` to the list of package topics (in addition to the existing `build-runner`) w/ the assumption that this package is often associated w/ those two topics.